models/lobby/configs_test.go

```diff
@@ -0,0 +1,62 @@
+package lobby
+
+import (
+	"log"
+	"testing"
+
+	"github.com/TeamPlayTF/Server/config"
+	"github.com/TeamPlayTF/Server/models/lobby"
+)
+
+func TestInitConfigs(t *testing.T) {
+	config.SetupConstants()
+	lobby.InitConfigs()
+}
+
+func TestUgcHighlander(t *testing.T) {
+	config := lobby.NewServerConfig()
+	config.League = lobby.LeagueUgc
+	config.Type = lobby.LobbyTypeHighlander
+	config.Map = "cp_process_final"
+	_, cfgErr := config.Get()
+
+	if cfgErr != nil {
+		log.Fatal(cfgErr)
```

Comment:
We're using https://github.com/stretchr/testify in our other tests so while this is ok, keep in mind you can (and probably should in the future) use asserts from that library.